File: db/src/main/java/com/readrops/db/pojo/FolderWithFeedCount.java
package com.readrops.db.pojo;

import androidx.room.ColumnInfo;
import androidx.room.Embedded;

import com.readrops.db.entities.Folder;

public class FolderWithFeedCount {

    @Embedded
    private Folder folder;

    @ColumnInfo(name = "feed_count")
    private int feedCount;

    public FolderWithFeedCount() {
    }

    public Folder getFolder() {
        return folder;
    }

    public void setFolder(Folder folder) {
        this.folder = folder;
    }

    public int getFeedCount() {
        return feedCount;
    }

    public void setFeedCount(int feedCount) {
        this.feedCount = feedCount;
    }
}

File: db/src/main/java/com/readrops/db/pojo/FolderWithFeedCount.kt
package com.readrops.db.pojo

import androidx.room.ColumnInfo
import androidx.room.Embedded
import com.readrops.db.entities.Folder

data class FolderWithFeedCount(
        @Embedded val folder: Folder,
        @ColumnInfo(name = "feed_count") val feedCount: Int,
)